liquidity-module/index.md

```diff
@@ -303,50 +308,51 @@ Add `username`s mnemonic passphrase and hit enter. You will see something simila
 
 ## Create a Liquidity Pool
 
-To create a liquidity pool, enter the following command:
-Make sure to replace `longibchash` with the hash denom you received on the previous step. Replace `username` with your account username.
+To: create a liquidity pool with the `gaiad tx liquidity create-pool` command:
 
 ```bash
-gaiad tx liquidity create-pool 1 1100000uphoton,1500000ibc/longibchash --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 --gas 2000000 -y
+gaiad tx liquidity create-pool 1 1100000uphoton,1500000ibc/longibchash --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 --gas 2000000
 ```
-**Optional:** Add -y flag to bypass confirmation prompt
 
-Confirm the pool has been created. See:
+For this example command, be sure to:
+
+- Replace `longibchash` with the hash denom that you received on the previous step
+- Replace `username` with your account username
+
+To confirm the pool has been created:
 
-[https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools](https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools)
+- Visit [https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools](https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools)
 
 
 ## Swap Token
 
 You are ready to swap tokens! You now have uphoton token in your account and want to swap for the new IBC coin:
 
 ```bash
-gaiad tx liquidity swap 1 1 100000uphoton ibc/longibchash 0.1 0.003 --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 -y
```

Comment:
Why not? 😅

Reply:
the theory behind optional steps in tutorials is that we want to provide the core experience for our users, no need to add in optional steps since it does not add to learning and just adds one more teensy bit of noise to explain

Reply:
Agreed @barriebyron But I think we should also find a way to encourage people to look at different flags and what it does. Any suggestions how can we do that?

Reply:
I always want to know what the flags do, too! 
Maybe an optional step? 
where are the flags documented? In the help and Starport CLI reference https://docs.starport.network/cli/ we really don't expose what the flags are. Teach me!